CMakeLists.txt

```diff
@@ -484,49 +484,47 @@ if(AVIF_CODEC_LIBRARIES MATCHES vmaf)
 endif()
 
 option(AVIF_BUILD_APPS "Build avif apps." OFF)
-if(AVIF_BUILD_APPS)
+option(AVIF_BUILD_TESTS "Build avif tests." OFF)
+
+if(AVIF_BUILD_APPS OR AVIF_BUILD_TESTS)
+    # Not all targets depending on avifappsshared, such as tests, need png and jpeg.
+    # However it is simpler this way and likely that a test environment provides these.
     find_package(ZLIB REQUIRED)
     find_package(PNG REQUIRED)
     find_package(JPEG REQUIRED)
 
-    add_executable(avifenc
-        apps/avifenc.c
-
+    add_library(avifappsshared STATIC # Static so it does not have to be installed.
```

Comment:
The comment "Static so it does not have to be installed." is confusing, because we can install a static library, and we don't need to install all shared libraries. I think we can omit this comment, or change it to say something like "For internal use only. Not installed."

Reply:
If `apps_apps` is shared and `avifenc` is installed, `apps_apps` must be installed too, right?

Reply:
Thanks for the explanation. I understand the comment now and have changed it back. Sorry about my confusion. I only considered the case of installing a library (including its headers) for external use (often called a -dev package).